helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -22,6 +22,13 @@ rules:
       - create
       - delete
       - get
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
```

Comment:
how come?

Reply:
Because deployment are in `apps` api group now

```
$ k api-resources
NAME                              SHORTNAMES   APIGROUP                       NAMESPACED   KIND
...
controllerrevisions                            apps                           true         ControllerRevision
daemonsets                        ds           apps                           true         DaemonSet
deployments                       deploy       apps                           true         Deployment
replicasets                       rs           apps                           true         ReplicaSet
statefulsets                      sts          apps                           true         StatefulSet
```